src/cls/2pc_queue/cls_2pc_queue.cc

```diff
@@ -113,6 +113,7 @@ static int cls_2pc_queue_reserve(cls_method_context_t hctx, bufferlist *in, buff
   std::tie(last_reservation, result) = urgent_data.reservations.emplace(std::piecewise_construct,
           std::forward_as_tuple(urgent_data.last_id),
           std::forward_as_tuple(res_op.size, ceph::coarse_real_clock::now()));
+  urgent_data.reservations[urgent_data.last_id].entries = res_op.entries;
```

Comment:
probably need to add that in line 152 as well

Reply:
Done